docker-compose.yml

```diff
@@ -76,3 +76,15 @@ services:
         depends_on:
             - metalnx-db
             - irods-catalog-provider
+
+    irods-client-nfsrods:
+        #image: irods/nfsrods:2.1.0
+        build:
+            context: irods_client_nfsrods
+        ports:
+            - "2049:2049"
```

Comment:
I wonder if we should use a different external port?

After rebooting my computer, I always have to disable the nfs-kernel-server service because it binds 2049.

Perhaps that indicates I should completely disable that service. Still, it's probably a good idea to use something like 2050 so we decrease the chance of others encountering that issue.

Reply:
Will change to 2050